dev-packages/cli/src/theia.ts

```diff
@@ -215,7 +216,10 @@ function rebuildCommand(command: string, target: ApplicationProps.Target): yargs
                 testSpec: string[],
                 testCoverage: boolean
             }) => {
+                let exitCode: number | undefined;
+                temp.track();
                 try {
+                    process.env.THEIA_CONFIG_DIR = process.env.THEIA_CONFIG_DIR || temp.mkdirSync('theia-test-config-dir');
```

Comment:
Minor: cleanup tmp directory when done. At least on Linux it will be cleared eventually, at the next reboot.